code/modules/research/techweb/all_nodes.dm

```diff
@@ -1456,7 +1456,10 @@
 	prereq_ids = list("adv_engi", "weaponry")
 	design_ids = list(
 		"pin_loyalty",
-		"ammo_workbench", //SKYRAT EDIT ADDITION
+		//SKYRAT EDIT START - RESEARCH DESIGNS
+		"smartdartrevolver",
```

Comment:
If nothing else it should be in its own node past this node, that also requires advanced biotech. It makes little sense sharing a node with general adv. weaponry.

Arguably it should be in Illegal with the RSG but /shrug

Reply:
Yes, but also the RSG can literally just function the exact same way as this but with the capability to load piercing syringes and standard syringes. It is a direct upgrade. That's why I feel it would be pointless to have the SmartDart revolver be on the same node as it.

Reply:
If a minor upgrade to a gun is so overpowered that it needs to be illegal tech, should the current grade really be avaable for anything less?

Reply:
Yes, the SmartDart revolver is a worse item, it should be available for less. SmartDarts themselves already drastically cut down the options that you have to harm people. (Not saying it's impossible to harm people with them, but you have to be creative or use more non-lethal knockout chems)

Reply:
Slightly less hilariously overpowered is still hilariously overpowered